benchs/gemm/gemm_bench_NVIDIA_A100_80GB_PCIe.csv

```diff
@@ -0,0 +1,14 @@
+M,N,K,kTM,kTN,kTK,cuBLAS(ms),Cutlass(ms),TiledCUDA(ms)
+4096,4096,2048,128,256,64,0.27764737606048584,1.56727294921875,nan

```

Comment:
Format the output to truncate to the given precision, such as 4 decimal places. Long numbers are inconvenient to read.